### DISCUSSION TITLE
Why <collection>.id is not present in generated schema

### DISCUSSION THREAD

**Discussion by Davidvision** [teunmooij/payload-swagger#74] [Q&A]
Hello ! I've successfully generated the OpenAPI specs but for some reason the "id" field, is not present in the generated schemas...
Did I miss something ?
see below, the "countries" schema doesn't include the id field, and the other ones neither... 😢 
<img width="330" alt="Capture d’écran 2023-10-19 à 20 19 43" src="https://github.com/teunmooij/payload-tools/assets/45124558/e840a6a8-4849-401f-b98d-c02c734ac607">

**Comment by jhnsml** [comment]
I've the same issue

**Comment by jhnsml** [comment]
@teunmooij Any workarounds for this?
